minidump-processor/src/process_state.rs

```diff
@@ -784,6 +854,15 @@ Unknown streams encountered:
                 "description": lsb.description,
             })),
             // optional
+            "proc_limits": self.linux_proc_limits.as_ref().map(|limits| json!({
```

Comment:
I am not 100% sure of the output we want to get here. Currently, this is what we have:
```
        "name": "Max file size",                                                                                                                                                                                                                                                                                                                        [54/1963]
        "soft": "Unlimited",
        "unit": "bytes"
      },
      {
        "hard": "Unlimited",
        "name": "Max resident set",
        "soft": "Unlimited",
        "unit": "bytes"
      },
      {
        "hard": {
          "Limited": 819200
        },
        "name": "Max msgqueue size",
        "soft": {
          "Limited": 819200
        },
        "unit": "bytes"
      },
      {
        "hard": {
          "Limited": 0
        },
        "name": "Max nice priority",
        "soft": {
          "Limited": 0
        },
        "unit": "n/a"
      },
      {
        "hard": {
          "Limited": 111064
        },
        "name": "Max processes",
        "soft": {
          "Limited": 111064
        },
        "unit": "processes"
      },
      {
        "hard": {
          "Limited": 1048576
        },
        "name": "Max open files",
        "soft": {
          "Limited": 1048576
        },
        "unit": "files"
      },
      {
        "hard": {
          "Limited": 3653476352
        },
        "name": "Max locked memory",
        "soft": {
          "Limited": 3653476352
        },
        "unit": "bytes"
      },
      {
        "hard": "Unlimited",
        "name": "Max address space",
        "soft": "Unlimited",
        "unit": "bytes"
      },
      {
        "hard": "Unlimited",
        "name": "Max stack size",
        "soft": {
          "Limited": 8388608
        },
        "unit": "bytes"
      }
```

Maybe we want a more specialized serde serializer and avoid indirections like `hard.Limited=xx` and just have `hard=xx` ?